src/prodbg/main/src/windows.rs

```diff
@@ -150,6 +175,7 @@ impl Windows {
                   sessions: &mut Sessions,
                   view_plugins: &mut ViewPlugins,
                   backend_plugins: &mut BackendPlugins) {
+        sleep(Duration::from_millis(100));
```

Comment:
Please tell me, do you need this code to inspect it (to make sure it does
what you want and how you want) or to use it in master right now (in which
case I'll remove all the debug code right now)?

2016-07-04 18:34 GMT+08:00 Daniel Collin <notifications@github.com>:

> In src/prodbg/main/src/windows.rs
> <https://github.com/emoon/ProDBG/pull/218#discussion_r69439121>:
>
> > @@ -150,6 +175,7 @@ impl Windows {
> >                    sessions: &mut Sessions,
> >                    view_plugins: &mut ViewPlugins,
> >                    backend_plugins: &mut BackendPlugins) {
> > +        sleep(Duration::from_millis(100));
>
> Agreed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/emoon/ProDBG/pull/218/files/d4db4aed4c9f97d6991801a031c077eb57d239f0#r69439121>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANveOk1bucY5K-Ex3uXTxlLk07b1p8Sks5qSOGrgaJpZM4JELvA>
> .
>